src/clr-angular/color/_variables.color.scss

```diff
@@ -1,62 +1,226 @@
+$clr-color-on-danger-300: $clr-color-neutral-1000 !default;
+$clr-color-on-danger-400: $clr-color-neutral-1000 !default;
+$clr-color-on-danger-500: $clr-color-neutral-1000 !default;
+$clr-color-on-danger-600: $clr-color-neutral-1000 !default;
+$clr-color-on-danger-700: $clr-color-neutral-0 !default;
+$clr-color-on-danger-800: $clr-color-neutral-0 !default;
+$clr-color-on-danger-900: $clr-color-neutral-0 !default;
+$clr-color-on-danger-1000: $clr-color-neutral-0 !default;
+
+// Stoplight Warning color: alerts, warning messages
+
+$clr-color-warning-50: #fffce8 !default;
+$clr-color-warning-100: #fef3b5 !default;
+$clr-color-warning-200: #ffe860 !default;
+$clr-color-warning-300: #ffdc0b !default;
+$clr-color-warning-400: #fdd006 !default;
+$clr-color-warning-500: #fac400 !default;
+$clr-color-warning-600: #edb200 !default;
+$clr-color-warning-700: #dfa100 !default;
+$clr-color-warning-800: #d28f00 !default;
+$clr-color-warning-900: #c47d00 !default;
+$clr-color-warning-1000: #601500 !default;
+
+$clr-color-on-warning-50: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-100: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-200: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-300: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-400: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-500: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-600: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-700: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-800: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-900: $clr-color-neutral-1000 !default;
+$clr-color-on-warning-1000: $clr-color-neutral-0 !default;
+
+// Stoplight Success color: alerts, success messaging, success buttons
+
+$clr-color-success-50: #dff0d0 !default;
+$clr-color-success-100: #c7e59c !default;
+$clr-color-success-200: #aadb1e !default;
+$clr-color-success-300: #85c81a !default;
+$clr-color-success-400: #60b515 !default;
+$clr-color-success-500: #62a420 !default;
+$clr-color-success-600: #48960c !default;
+$clr-color-success-700: #2f8400 !default;
+$clr-color-success-800: #266900 !default;
+$clr-color-success-900: #1d5100 !default;
+$clr-color-success-1000: #0d4100 !default;
+
+$clr-color-on-success-50: $clr-color-neutral-1000 !default;
+$clr-color-on-success-100: $clr-color-neutral-1000 !default;
+$clr-color-on-success-200: $clr-color-neutral-1000 !default;
+$clr-color-on-success-300: $clr-color-neutral-1000 !default;
+$clr-color-on-success-400: $clr-color-neutral-1000 !default;
+$clr-color-on-success-500: $clr-color-neutral-1000 !default;
+$clr-color-on-success-600: $clr-color-neutral-1000 !default;
+$clr-color-on-success-700: $clr-color-neutral-0 !default;
+$clr-color-on-success-800: $clr-color-neutral-0 !default;
+$clr-color-on-success-900: $clr-color-neutral-0 !default;
+$clr-color-on-success-1000: $clr-color-neutral-0 !default;
+
+// DEPRECATED COLOR VARIABLES
```

Comment:
Easiest way to do that is to use a function. The problem is that the variables themselves are over-rideable. So if they are overridden, it won't use that function. It may be enough to update the 2.0 docs after this is merged in.